Scenes/BottomBar/Skills/SpeedupBtn.gd

```diff
@@ -0,0 +1,29 @@
+extends "res://Scenes/BottomBar/Skills/SkillBtn.gd"
+
+onready var world = get_node("/root/World")
+onready var level_manager = world.find_node("LevelManager")
+
+
+func use() -> void:
+	remove_nodes_in_group("IDwarf")
+	remove_nodes_in_group("IArrow")
+	jump_to_next_boss_level()
+	spawn_boss()
+	.use()
+	
+func remove_nodes_in_group(group_name:String) -> void:
+	var nodes = get_tree().get_nodes_in_group(group_name)
+	
+	for node in nodes:
+		node.queue_free()
+	
+func jump_to_next_boss_level() -> void:
+	var current_level : int = level_manager.current_level
+	var next_boss_level : int = (floor(current_level / 10)+1) * 10
```

Comment:
Maybe extract it into the new method in ```dwarves_spawner```:
```var next_boss_level : int = dwarves_spawner.return_next_boss_level()```

Reply:
Remember, We're going to change the name of the class ```dwarves_spawner``` to ```dwarvers_manager``` and this method, in ```dwarvers_manager``` will be right

Reply:
That's good idea. But I made this function in LevelManager in this PR, because might be situation that game should spawn more than one boss while using one SpeedUp skill. You can check this in LevelManager.gd line 89. So I just used this function in current fixing commit.

Reply:
Ok